dubbo-admin-server/pkg/model/registry_source.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type RegistrySource int
+
+const (
+	ALL RegistrySource = iota
```

Comment:
The constants should follow the Camel case. Thus the `ALL` should be `All`, and so on.